oceannavigator/frontend/src/components/Map.jsx

```diff
@@ -170,6 +170,10 @@ export default class Map extends React.PureComponent {
       location: [0, 90],
     };
 
+    this.scaleViewer = new app.ScaleViewer({
+      image: "/api/v1.0/scale/giops_day/votemper/-5,30.png",
+    });
```

Comment:
Create a default colourmap bar when page first loads using default dataset and variable.